Controls/Tests/Test_Cafe.c

```diff
@@ -22,8 +22,55 @@ int main(void){
 
 	// Create any semaphores/mutexes and initialize any global variables here
 	
+	OSMutexCreate(
+		&RegisterOccupied_Mutex,
+		"Mutex",
+		&err
+	);
 
 	// Initialize both tasks here
+
+	OSTaskCreate(
+		(OS_TCB*) &Customer_1_TCB, 
+		(CPU_CHAR*) "Customer 1 Failed... \n",
```

Comment:
This is technically the name for the Customer 1 task, not an error message. You might just wanna call this "Customer 1".